docs/docs/docs/sbt/index.md

```diff
@@ -39,6 +41,9 @@ The configuration works as follows:
    `ProtocolAndVersion` provides two values: `name` (mandatory) that
    corresponds with the identifier used to store the protocol and
    `version` (optional). Default: Nil
+-  `compendiumSrcGenProtobufCompressionType`: *CompressionTypeGen*. Only for protobuf schema. Supported: `NoCompressionGen`, `GzipGen`.
```

Comment:
I'm curious @mrtmmr, the compression is supported by avro too, what are the reasons to support only proto?

Reply:
There's no strong reason. We needed to start from somewhere and this was the first step. This changed was suggested in a certain context where there was a person available to do small improvements like this.